doc/proposals/ansible-helm-addapi.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Ansible/Helm add api command  Proposal for Operator SDK
+authors:
+  - "@bharathi-tenneti"
+reviewers:
+  - "@fabianvf"
+  - "@cmacedo"
+approvers:
+  - "@fabianvf"
+  - "@jlanford"
+  - "@dmesser"
+creation-date: 2020-03-10
+last-updated: 2020-03-11
+status: implementable
+---
+
+
+# Ansible/Helm add api command  Proposal for Operator SDK
+
+
+## Release Signoff Checklist
+
+- \[x\] Enhancement is `implementable`
+- \[x\] Design details are appropriately documented from clear requirements
```

Comment:
I think it would be just checked if the proposal has detailed information over how to do the implementation.
For example, is possible to use the operator-sdk api command that already exists today? When we call the operator-sdk new <project> where is this implementation? Can we use it in the operator-sdk api? Will be required to implement a specific command for Helm and other for Ansible or not? 

Also, I'd add the info regards how we can check the type of the operator whiout the need to use the flag --type

Reply:
"Also, I'd add the info regards how we can check the type of the operator whiout the need to use the flag --type"
For above comment, The idea is to modify "add api" command, which does not look for --type flag, rather use [this](https://github.com/operator-framework/operator-sdk/blob/master/cmd/operator-sdk/new/cmd.go#L121) to determine the operator type.